src/js/game/LevelMVC/LevelView.js

```diff
@@ -1650,6 +1738,11 @@ module.exports = class LevelView {
       sprite = null,
       frameList;
 
+    // Need to make sure the switch case will capture the right miniBlock for -all- redstoneWire
+    if (blockType.substring(0,12) === "redstoneWire") {
+      blockType = "redstoneWire";
```

Comment:
Is this right?  Now that we have `redstoneWireHorizontal` etc. I think we want that as the block type?

Reply:
Ah, I see — this is for dropping mini blocks on destroy.